packages/js/block-templates/src/utils/register-woo-block-type.ts

```diff
@@ -55,8 +58,8 @@ function getEdit<
 		const { getEvaluationContext } = useEvaluationContext( context );
 
 		const { shouldHide, shouldDisable } = useSelect(
-			// @ts-expect-error TODO: react-18-upgrade
-			( select: typeof WPSelect ) => {
+			// eslint-disable-next-line @typescript-eslint/no-explicit-any
+			( select: any ) => {
```

Comment:
You've already declared the generic SelectType. Perhaps we can use that here so we don't need to use `any`?

Reply:
Nice catch. Updated in b0964de.